internal/common/db.go

```diff
@@ -76,6 +76,13 @@ type Host struct {
 	TriggerMonitorTimestamp time.Time
 }
 
+type InfraEnv struct {
+	models.InfraEnv
+
+	// The pull secret that obtained from the Pull Secret page on the Red Hat OpenShift Cluster Manager site.
+	PullSecret string `json:"pull_secret" gorm:"type:TEXT"`
```

Comment:
we need to add KubeKeyNamespace string `json:"kube_key_namespace"` as a preparation for creating InfraEnv via specific API